client/finality-grandpa/src/lib.rs

```diff
@@ -84,6 +84,28 @@ use std::time::Duration;
 use std::pin::Pin;
 use std::task::{Poll, Context};
 
+#[macro_use]
+mod macros {
+	// utility logging macro that takes as first argument a conditional to
+	// decide whether to log under debug or info level (useful to restrict
+	// logging under initial sync).
+	// defined in a separate module since we want to share the macro across this
+	// crate but not export it.
```

Comment:
Only macros tagged with `#[macro_export]` are exported, so the extra module is not needed

Reply:
Yeah, I was being dumb. The reason this didn't work initially (without the mod and macro_use) was just because I had defined the macro later than the modules that were using it.